pkg/storage/streams/store.go

```diff
@@ -154,6 +176,8 @@ func (s *streamStore) Get(ctx context.Context, path paths.Path) (
 		return nil, Meta{}, err
 	}
 
+	encKey := sha256.Sum256([]byte(s.key))
+	var firstNonce [12]byte
 	var rangers []ranger.RangeCloser
 
 	for i := 0; i < int(msi.NumberOfSegments); i++ {
```

Comment:
Perhaps, initialize `i := int64(0)` and this will help get rid of some strange type casts.